src/components/Login/Login.js

```diff
@@ -12,11 +13,13 @@ import styles from './Login.scss';
 const TAB_WIF = 'wif';
 const TAB_PASSPHRASE = 'passphrase';
 const TAB_LEDGER = 'ledger';
+const TAB_JSONFILE = 'jsonfile';
 
 const TABS = {
   [TAB_PASSPHRASE]: 'Passphrase',
   [TAB_WIF]: 'WIF',
-  [TAB_LEDGER]: 'Ledger'
+  [TAB_LEDGER]: 'Ledger',
+  [TAB_JSONFILE]: 'Json File'
```

Comment:
I think we should call this "Wallet File" for clarity.

Reply:
OK, changed locally